SearchInfiniteArray.java

```diff
@@ -0,0 +1,46 @@
+//time - O(log n)
+//space - O(1)
+
+//please help me with the case when high goes out of bounds, I was trying to use a try catch block but couldn't complete it
```

Comment:
https://github.com/devashishnyati/Binary-Search-1/blob/master/unkownlength.py
Look at this code. Let me know if you need any help.